### ISSUE TITLE
[wled] Add channel for power consumption

### ISSUE THREAD

**Issue by TheNetStriker** [openhab/openhab-addons#15670]
It would be nice to have a brigde channel with the wled total power consumption.

The field is located in the json under **json/info** in the field **leds.pwr**.

## Your Environment
* Version used: OpenHab 4.0.3
* wled version: 0.14.0-b6

**Comment by Skinah** [comment]
I'll look next time I am making changes to add this. Just want to confirm you understand this is a faked guessed value and not a measured one? At least it was when I read this in the WLED website. I did not bother to add it for this reason, but am happy to add it since someone wants it.

**Comment by TheNetStriker** [comment]
> I'll look next time I am making changes to add this. Just want to confirm you understand this is a faked guessed value and not a measured one? At least it was when I read this in the WLED website. I did not bother to add it for this reason, but am happy to add it since someone wants it.

Yes, I'm aware that this value is just a guess, but it would still be nice to have this value to see what power consumpton wled esimates.

**Comment by TheNetStriker** [comment]
@Skinah Any updates regarding this feature request?

If you don't have the time for this I could take a look at it and create a pull request.
